app/routes/models/application-history.js

```diff
@@ -8,72 +8,71 @@ const formatDate = (dateToFormat, currentDateFormat = 'YYYY-MM-DD', dateFormat =
   return ''
 }
 
-const parseData = (payload, key) => {
-  const data = payload ? JSON.parse(payload) : {}
-  return data[key]
-}
-
-const filterRecords = (applicationHistory) => {
-  return [
-    ...(applicationHistory.historyRecords?.filter(apphr =>
-      [
-        applicationStatus.withdrawn,
-        applicationStatus.readyToPay,
-        applicationStatus.rejected,
-        applicationStatus.onHold,
-        applicationStatus.inCheck,
-        applicationStatus.recommendToPay,
-        applicationStatus.recommendToReject
-      ].includes(parseData(apphr.Payload, 'statusId'))
-    ) || [])
-  ]
-}
+const filterRecords = (applicationHistory) => applicationHistory
+  .historyRecords.filter((apphr) =>
+    [
+      applicationStatus.withdrawn,
+      applicationStatus.readyToPay,
+      applicationStatus.rejected,
+      applicationStatus.onHold,
+      applicationStatus.inCheck,
+      applicationStatus.recommendToPay,
+      applicationStatus.recommendToReject
+    ].includes(JSON.parse(apphr.Payload).statusId)
+  )
 
 const getStatusText = (status, subStatus) => {
   switch (status) {
     case applicationStatus.withdrawn:
-      return 'Withdraw completed'
+      return 'Withdrawn'
```

Comment:
What was the thinking behind updating these labels? Was this something EAT team have expressed an interest in?

Reply:
I was just trying to get them to make a bit more sense 🤷. 'Withdraw completed' didn't feel like clear unambiguous language.

Changing 'Claim approved' & 'Claim rejected' to 'Approved' & 'Rejected' was removing a superfluous noun. It's obviously a claim.

The others I just tried to make consistent. We could probably get rid of the 'Moved to' prefixes but didn't want to change it too much without talking to anyone.

Feel it's a bit clearer now but could be improved further.